portfolio/src/main/webapp/script.js

```diff
@@ -28,3 +28,30 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function getHello() {
+    // const response = await fetch('/data');
+    // const quote = await response.text();
+    // document.getElementById('quote-container').innerText = quote;
+    fetch('/data')
+    .then(response => response.json())
+    .then((funFacts) => {
+        const myStrings = document.getElementById('quote-container');
```

Comment:
It might also be good to add a check here that the response contains 3 elements. Otherwise, if I understand correctly, this will currently fail with an index out of bounds error.

Reply:
Interesting, how would I error-handle the response? Is there a way to check the length of the response maybe?

Reply:
Does funFacts have a field, "length" that you could use?

Reply:
Perfect, I found the .length field and used it to throw an error if the length is incorrect. Thank you!